.rubocop.yml

```diff
@@ -138,6 +138,8 @@ Style/StringLiterals:
   EnforcedStyle: double_quotes
 Style/StringLiteralsInInterpolation:
   EnforcedStyle: double_quotes
+Style/SymbolArray:
+  Enabled: false
```

Comment:
> this needs to be shipped as a unique pull request for everyone else to comment.

This PR will ship with this cop `disabled` in line with behavior from `0.47.1` It can `enabled` via a dedicated PR